org.eclipse.xtext/src/org/eclipse/xtext/formatting2/debug/TextRegionAccessToString.java

```diff
@@ -293,7 +305,7 @@ public boolean isHideIndentation() {
 	}
 
 	public boolean isHightlightOrigin() {
-		return hightlightOrigin;
+		return highlightOrigin;
```

Comment:
do we need a new version deprecation here too?

Reply:
We do, didn't notice that one

Reply:
Done